assets/xml/scenes/indoors/tokinoma.xml

```diff
@@ -2,6 +2,7 @@
     <File Name="tokinoma_scene" Segment="2">
         <Cutscene  Name="gFirstAdultCs" Offset="0x46F0"/>
         <Cutscene  Name="gPreludeCs" Offset="0x6D20"/>
```

Comment:
not your change but since you're introducing scene prefixes, maybe those should use them as well